packages/cli/src/commands/env/pull.ts

```diff
@@ -117,7 +117,8 @@ export default async function pull(
   if (exists) {
     oldEnv = await createEnvObject(fullPath, output);
     if (oldEnv) {
-      deltaString = buildDeltaString(oldEnv, records);
+      const newEnv = JSON.parse(JSON.stringify(records).replace(/\\"/g, ''));
```

Comment:
The intent is to remove all `"` from within strings? Since that's specific to building the delta string, perhaps this should go in `buildDeltaString()`?

Reply:
Possibly, but I think it would be more consistent to keep it outside. The quotes etc. from `oldEnv` are removed in `createEnvObject()`, which is also called before. I think it would be strange to move it to `buildDeltaString()`. But I'm happy to do it if we feel it's better.

Reply:
OK, makes sense. If `createEnvObject()` strips the quotes, then would it be consistent if `exposeSystemEnvs()` did the same?

Reply:
I'm concerned that will have unintended side effects that will be difficult to determine. The point of the manipulation here is just to get each set of environment variables in a state where they can both be compared, but `exposeSystemEnvs()` is used in other places in the code that may expect to receive environment variables with no manipulations.

I don't know too much about how this part of Vercel works to feel confident in this, though.